sites/main-page-title/index.json-ld

```diff
@@ -354,11 +418,29 @@
 			],
 			"predecessor": "_:iteration_mpt_2"
 		},
+		{
+			"@id": "_:iteration_mpt_4",
+			"name": "Main page title aligned on the right",
+			"date": "2024-09",
+			"detectableBy": "h1#wb-cont[dir='rtl'], [dir='rtl'] h1#wb-cont",
+			"additions": [
+				"Added right to left support"
+			]
+		},
 		{
 			"@id": "_:iteration_smpt_1",
 			"name": "Stacked main page title - Iteration 1",
 			"date": "2024-08",
 			"detectableBy": "hgroup > h1#wb-cont"
+		},
+		{
+			"@id": "_:iteration_smpt_2",
+			"name": "Stacked main page title - Iteration 2 (LTR support)",
+			"date": "2024-10",
+			"detectableBy": "hgroup > h1#wb-cont[dir='ltr'], [dir='ltr'] hgroup > h1#wb-cont",
+			"additions": [
+				"Added right to left support"
+			]
```

Comment:
Please also add `"successor": "_:iteration_smpt_2"` to the Iteration 1.
```suggestion
			],
			"predecessor": "_:iteration_smpt_1"
```

Reply:
Added. Thanks